dataland-frontend/src/components/forms/parts/elements/derived/ProductFormElement.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div data-test="productFormElement">
+    <InputTextFormField
+      name="productName"
+      label="Product Name"
+      description="Please enter the name of the product"
+      :required="true"
+      validation="required"
+    />
+    <StringBadgeListFormField
+      name="productionSteps"
+      label="Production Steps"
+      description="Please give a brief overview of the production steps/activities undertaken"
+      data-test-add-button="addProductionStep"
+    />
+    <FreeTextFormField
+      name="relatedCorporateSupplyChain"
+      label="Related Corporate Supply Chain"
+      description="Please give an overview of the related corporate supply chain(s) and key business relationships (by procurement or order volume) (own operations)"
+      v-model="existingRelatedCorporateSupplyChain"
```

Comment:
Why is this v-model required?

Reply:
I don't see the point. What about you @MarcinStanczyk7 ?

Reply:
neither do I. Removed

Reply:
👍